pkg/container/network.go

```diff
@@ -117,19 +117,24 @@ func bridge(iface *net.Interface) (*DHCPInterface, error) {
 	tapName := constants.TAP_PREFIX + iface.Name
 	bridgeName := constants.BRIDGE_PREFIX + iface.Name
 
-	if err := createTAPAdapter(tapName); err != nil {
+	handle, err := netlink.NewHandle()
+	if err != nil {
+		return nil, err
+	}
+
+	if err := createTAPAdapter(handle, tapName); err != nil {
 		return nil, err
 	}
 
-	if err := createBridge(bridgeName); err != nil {
+	if err := createBridge(handle, bridgeName); err != nil {
 		return nil, err
 	}
 
-	if err := connectAdapterToBridge(tapName, bridgeName); err != nil {
+	if err := connectAdapterToBridge(handle, tapName, bridgeName); err != nil {
 		return nil, err
 	}
 
-	if err := connectAdapterToBridge(iface.Name, bridgeName); err != nil {
+	if err := connectAdapterToBridge(handle, iface.Name, bridgeName); err != nil {
```

Comment:
Same as above, use `netlink.LinkByName(iface.Name)` to retrieve the device for `handle.LinkSetMaster`